### PR TITLE
Updates Infinispan to 14.0.11.Final

### PR DESCRIPTION
**Pull request by karesti**
Latest minor release

### CODE REVIEW

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
This native image failure looks related:

```
Error: Classes that should be initialized at run time got initialized during image building:
 io.netty.channel.unix.FileDescriptor the class was requested to be initialized at run time (from feature io.quarkus.runner.Feature.beforeAnalysis with 'FileDescriptor.class'). To see why io.netty.channel.unix.FileDescriptor got initialized use --trace-class-initialization=io.netty.channel.unix.FileDescriptor
io.netty.util.AbstractReferenceCounted the class was requested to be initialized at run time (from 'META-INF/native-image/io.netty/netty-common/native-image.properties' in 'file:///project/lib/io.netty.netty-common-4.1.94.Final.jar' with 'io.netty.util.AbstractReferenceCounted'). To see why io.netty.util.AbstractReferenceCounted got initialized use --trace-class-initialization=io.netty.util.AbstractReferenceCounted
io.netty.channel.epoll.Epoll the class was requested to be initialized at run time (from feature io.quarkus.runner.Feature.beforeAnalysis with 'Epoll.class'). To see why io.netty.channel.epoll.Epoll got initialized use --trace-class-initialization=io.netty.channel.epoll.Epoll
io.netty.channel.epoll.Native the class was requested to be initialized at run time (from feature io.quarkus.runner.Feature.beforeAnalysis with 'Native.class'). To see why io.netty.channel.epoll.Native got initialized use --trace-class-initialization=io.netty.channel.epoll.Native
To see how the classes got initialized, use --trace-class-initialization=io.netty.channel.unix.FileDescriptor,io.netty.util.AbstractReferenceCounted,io.netty.channel.epoll.Epoll,io.netty.channel.epoll.Native
com.oracle.svm.core.util.UserError$UserException: Classes that should be initialized at run time got initialized during image building:
 io.netty.channel.unix.FileDescriptor the class was requested to be initialized at run time (from feature io.quarkus.runner.Feature.beforeAnalysis with 'FileDescriptor.class'). To see why io.netty.channel.unix.FileDescriptor got initialized use --trace-class-initialization=io.netty.channel.unix.FileDescriptor
io.netty.util.AbstractReferenceCounted the class was requested to be initialized at run time (from 'META-INF/native-image/io.netty/netty-common/native-image.properties' in 'file:///project/lib/io.netty.netty-common-4.1.94.Final.jar' with 'io.netty.util.AbstractReferenceCounted'). To see why io.netty.util.AbstractReferenceCounted got initialized use --trace-class-initialization=io.netty.util.AbstractReferenceCounted
io.netty.channel.epoll.Epoll the class was requested to be initialized at run time (from feature io.quarkus.runner.Feature.beforeAnalysis with 'Epoll.class'). To see why io.netty.channel.epoll.Epoll got initialized use --trace-class-initialization=io.netty.channel.epoll.Epoll
io.netty.channel.epoll.Native the class was requested to be initialized at run time (from feature io.quarkus.runner.Feature.beforeAnalysis with 'Native.class'). To see why io.netty.channel.epoll.Native got initialized use --trace-class-initialization=io.netty.channel.epoll.Native
To see how the classes got initialized, use --trace-class-initialization=io.netty.channel.unix.FileDescriptor,io.netty.util.AbstractReferenceCounted,io.netty.channel.epoll.Epoll,io.netty.channel.epoll.Native
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:73)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ProvenSafeClassInitializationSupport.checkDelayedInitialization(ProvenSafeClassInitializationSupport.java:277)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationFeature.duringAnalysis(ClassInitializationFeature.java:164)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.lambda$runPointsToAnalysis$10(NativeImageGenerator.java:770)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.FeatureHandler.forEachFeature(FeatureHandler.java:86)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.lambda$runPointsToAnalysis$11(NativeImageGenerator.java:770)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.AbstractAnalysisEngine.runAnalysis(AbstractAnalysisEngine.java:179)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:767)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:582)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:539)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:408)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:612)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.start(NativeImageGeneratorRunner.java:134)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:94)
```

**Comment by geoand** [comment]
Very true. @karesti are you looking into it or should I?

**Comment by karesti** [comment]
weird

**Comment by karesti** [comment]
looking

**Comment by karesti** [comment]
@gsmet I'm looking with the team what's happening here, keep you posted

**Comment by karesti** [comment]
@geoand @gsmet I excluded a dependency related to netty and epoll, native and jvm seems to pass locally for me, finger crossed 
thanks for the heads up, I believed all was fine but probably I did not run all tests in native locally or did it during the other PR (that did not update infinispan...)

**Comment by karesti** [comment]
@gsmet @geoand looks ready

File: extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/NettySubstitution.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+final class NettySubstitution {
+}
+
+@TargetClass(value = NativeTransport.class)
+final class SubstituteNativeTransport {
+    @Substitute
+    private static boolean useNativeEpoll() {
+        return false;
+    }
+}
```
Comment (geoand) [inline extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/NettySubstitution.java:17]: We can probably improve this in the future, but let's leave it for now

**Pull request merged**